api/src/controllers/post/inspection.js

```diff
@@ -0,0 +1,178 @@
+var mongoose = require('mongoose');
+var ObjectId = require('mongoose').Types.ObjectId;
+
+// Example of incomingObj
```

Comment:
nitpick:  You might as well include this comment inside the /** */ comment block, rather than have it separated.